src/Microsoft.AspNet.Mvc.Xml/Formatters/XmlOutputFormatter.cs

```diff
@@ -73,5 +75,27 @@ public static XmlWriterSettings GetDefaultXmlWriterSettings()
         {
             return XmlWriter.Create(writeStream, xmlWriterSettings);
         }
+
+        private Type GetSerializableErrorType(Type type)
+        {
+            if (type == Constants.SerializableErrorType)
+            {
+                return Constants.SerializableErrorWrapperType;
+            }
+
+            return type;
+        }
+
+        internal object GetWrappedSerializableErrorObject(object obj)
```

Comment:
:smile: i  understand that still i think it should be protected.. leave it up for others to comment on the pattern here.